pep-0622.rst

```diff
@@ -318,24 +318,22 @@ Constant Value Patterns
 This is used to match against constants and enum values.
 Every dotted name in a pattern is looked up using normal Python name
 resolution rules, and the value is used for comparison by equality with
-the matching expression (same as for literals)::
+the match subject (same as for literals)::
 
   from enum import Enum
 
-  class Color(Enum):
-      BLACK = 1
-      RED = 2
-
-  BLACK = 1
-  RED = 2
+  class Sides(str, Enum):
+      SPAM = "spam"
+      EGGS = "eggs"
+      ...
 
-  match color:
-      case Color.BLACK:
-          print("Black suits every color")
-      case BLACK:  # This will just assign a new value to BLACK.
-          ...
+  match entree[-1]:
+      case Sides.SPAM:  # Compares entree[-1] == Sides.SPAM.
+          response = f"Have you got anything without {Sides.SPAM}?"
+      case side:  # Assigns side == entree[-1].
+          response = f"Well, could I have their {Sides.SPAM} instead of the {side} then?"
 
-Note that it is not possible for unqualified names (such as ``BLACK``
+Note that it is not possible for unqualified names (such as ``side``
 in the example above) to be used as constant value patterns.  See
```

Comment:
Somehow using `side` above feels distracting too. The original phrase was a bit awkward already. How about this?
```suggestion
Note that there is no way to use unqualified names as constant value
patterns (they always denote variables to be captured).  See
```